### PR TITLE
Printing the exit codes of the scripts we run when we test

### PR DESCRIPTION
**Pull request by zingero**
Now, we don't need to search the exit codes. After running the tests, it'll print a summary that looks like this:

> -------- Summary of exit codes --------
pycodestyle: 0
pylint: 0
unitests: 0

### CODE REVIEW

File: controller/scripts/entrypoint-test.sh [hunk 1]
```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 set -x
```
Comment (oriyarde) [inline controller/scripts/entrypoint-test.sh:2]: could you remove all `set` lines in this file? IMO prints like `+ PYCODESTYLE=0` are just noise

Comment (zingero) [reply controller/scripts/entrypoint-test.sh]: Done

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
NICE

**Pull request merged**